test/benchmarks_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:convert' show JsonEncoder;
+import 'dart:io';
+
+import 'package:test/test.dart';
```

Comment:
nit: reorder

Reply:
Reordered alphabetically. Is this correct?